src/crypto/kem/kem.go

```diff
@@ -0,0 +1,209 @@
+package kem
+
+import (
+	"circl/dh/sidh"
+	"circl/kem/schemes"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"io"
+
+	"golang.org/x/crypto/curve25519"
+)
+
+// ID identifies each flavor of KEM.
+type ID uint16
+
+const (
+	// KEM25519 is X25519 as a KEM. Not quantum-safe.
+	KEM25519 ID = 0x01fb
+	// CSIDH is a post-quantum NIKE
+	CSIDH ID = 0x01fc
```

Comment:
```suggestion
```
I probably initially included this because that's the "first" KEM supported by the Rustls stack; but it is not used elsewhere and breaks the validation logic with `minKEM`, `maxKEM`.

Reply:
Relatedly: the kems need to be renumbered.

Reply:
Yeah.. I'll change them ;)

Reply:
Should be better now.